docs/ref/file-binary.md

```diff
@@ -103,6 +108,15 @@ q)`:hello 1: 0x68656c6c6f776f726c64
 `:hello
 ```
 
+### Compression
+
+Since 4.1t 2023.04.17 data can be compressed while writting, by including [`compression parameters`](../kb/file-compression.md#compression-parameters)
```

Comment:
```suggestion
Since 4.1t 2023.04.17 data can be compressed while writing, by including [`compression parameters`](../kb/file-compression.md#compression-parameters)
```